### DISCUSSION TITLE
Not sure how to customize the bot?

### DISCUSSION THREAD

**Discussion by cubixty** [discord-tickets/bot#286] [Support (Q&A)] [answered]
Basically the title. Im new to this. Any help would be much appreciated

**Comment by eartharoid** [comment, marked as answer]
- Change the server's settings: `/settings set ...`
- Change a category's settings: `/settings categories edit id:... ....`

There are also some options in the config file (`user/config.js`).

**Comment by RooRay** [comment]
Following up on Earth's reply, other commands are also outlined [here](https://discordtickets.app/commands)
